src/ucan/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './types'
+export * from './token'
+export * as dictionary from './dictionary'
```

Comment:
we export helper methods for looking up ucans in dictionary, but not actual store itself (which is in `ucan/store.ts`)